scripts/gentest/src/main.rs

```diff
@@ -234,11 +234,17 @@ fn generate_assertions(ident: &str, node: &Value, use_rounding: bool) -> TokenSt
     let layout = if use_rounding { &node["smartRoundedLayout"] } else { &node["unroundedLayout"] };
 
     let read_f32 = |s: &str| layout[s].as_f64().unwrap() as f32;
+    let read_naive_f32 = |s: &str| node["naivelyRoundedLayout"][s].as_f64().unwrap() as f32;
     let width = read_f32("width");
     let height = read_f32("height");
     let x = read_f32("x");
     let y = read_f32("y");
 
+    // TODO: Use clientWidth/clientHeight to take into account scrollbars
+    // TODO: Handle rounding better
```

Comment:
There are some TODOs here, do they need to be resolved in this PR or is this for a different issue?

Reply:
Ah yes, that is to take into account the affect that scrollbars have on the scroll width/height (they take up space, so they actually make you scroll slightly further). That could either be done as part of this PR, or as a follow up PR.

Reply:
This has now been implemented